lib/rules/declaration-block-no-duplicate-properties/__tests__/index.js

```diff
@@ -114,6 +114,10 @@ testRule(rule, {
     {
       code: "a { -WEBKIT-border-radius: 12px; -webkit-BORDER-radius: 10px; }",
       message: messages.rejected("-webkit-BORDER-radius")
+    },
```

Comment:
Can we please pull this out into its own `testRule` block with `syntax: styled`, please?

We should place it at the end of the file, after the `syntax: html` `testRule` block.